src/main/java/cn/nukkit/inventory/Fuel.java

```diff
@@ -67,5 +67,6 @@
         duration.put(Item.ACACIA_DOOR, (short) 200);
         duration.put(Item.DARK_OAK_DOOR, (short) 200);
         duration.put(Item.BANNER, (short) 300);
+        duration.put(Item.DRIED_KELP_BLOCK, (short) 4000);
```

Comment:
This is actually registering a block id instead of item id, this can cause unexpected side-effects, like making POISONOUS_POTATO become a fuel too because it has the same numeric ID.

This can be avoided if you register it with negative id when the ID is higher than `255`.
```java
255 - BlockID.DRIED_KELP_BLOCK
```

Would be nice to create two new methods, one to register item and other to register block and replace that direct put calls with the call to the new method, this would make it safer and easier to maintain.

Reply:
I agree with you, the poison potato problem would really occur (but only if placed in the furnace through a hopper, since the client blocks it from being placed through the GUI, tested). I did both methods as requested, but the method of adding blocks still needs to be checked if the ID is greater than 255 because some fuel blocks have IDS less than 255 and greater than 0 (like music block, mushroom blocks and the coal block)

Reply:
@Gabriel8579 yes, it needs an if statement, you can use this method as reference: https://github.com/GameModsBR/PowerNukkit/blob/88f8392760b3ca4ce4ed615d4ae39cb2b708c63c/src/main/java/cn/nukkit/block/Block.java#L786-L793

Reply:
I put this if statement in the Fuel class method to avoid incompatibility problems